<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1228</width>
    <height>735</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>14</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Aplikasi Absensi Wajah</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>10</y>
     <width>601</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(94, 92, 100);</string>
   </property>
   <property name="text">
    <string>APLIKASI ABSENSI WAJAH</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="dataKaryawanButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>340</y>
     <width>251</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>Data Karyawan</string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../assets/4.png</normaloff>../assets/4.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>50</width>
     <height>50</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="dataJadwalButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>270</y>
     <width>251</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>Data Jadwal</string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../assets/3.png</normaloff>../assets/3.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>50</width>
     <height>50</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="absensiWajahButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>251</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>Absen Wajah</string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../assets/1.png</normaloff>../assets/1.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>50</width>
     <height>50</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="dataAbsensiButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>200</y>
     <width>251</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>Data Absensi</string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../assets/2.png</normaloff>../assets/2.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>50</width>
     <height>50</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>91</width>
     <height>91</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../assets/logo.jpeg</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>50</y>
     <width>631</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Jl. Gatot Subroto No.27, RT.02, Sungai Asam, Kec. Ps. Jambi,Kota Jambi, Jambi 36123</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>70</y>
     <width>491</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>082377399933</string>
   </property>
  </widget>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>120</y>
     <width>941</width>
     <height>601</height>
    </rect>
   </property>
   <widget class="QWidget" name="page"/>
   <widget class="QWidget" name="page_2"/>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
